### PR TITLE
🌐 Add Traditional Chinese translation for `docs/zh-hant/docs/async.md`

### PR DESCRIPTION
**Pull request by ILoveSorasakiHina**

### CODE REVIEW

**Review by mattwang44 — APPROVED** [review]
i've reviewed in #12489

File: docs/zh-hant/docs/async.md [hunk 1]
```diff
@@ -0,0 +1,442 @@
...
+
+
+```Python hl_lines="2"
+@app.get('/')
+async def read_results():
+    results = await some_library()
+    return results
+```
+
+/// note | 注意
```
Comment (alejsdev) [inline docs/zh-hant/docs/async.md:25]: I've fixed this (it wasn't rendering properly)
(we have to include the “note” as the styling depends on this part)

**Comment by alejsdev** [comment]
Thank you! @ILoveSorasakiHina :rocket: 
And thanks for your (previous) review @mattwang44 :nerd_face:

**Review by Kagney00 — COMMENTED** [review]
Here

**Pull request merged**